sdk/synapse/azure-analytics-synapse-accesscontrol/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2021-04-06)
 
+- `listRoleAssignmentsWithResponse()` now returns `RoleAssignmentsListRoleAssignmentsResponse`
+- Update azure-core to 1.15.0
```

Comment:
Group these updates into "Breaking Changes", "Dependency Updates", "New features" or "Bug fixes" in all the CHANGELOGS in this PR.
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md#700-beta7-2020-11-06

Reply:
Grouped